plugins/source/aws/resources/services/s3/bucket_cors_rules.go

```diff
@@ -22,9 +22,10 @@ func bucketCorsRules() *schema.Table {
 		Columns: []schema.Column{
 			client.DefaultAccountIDColumn(false),
 			{
-				Name:     "bucket_arn",
-				Type:     arrow.BinaryTypes.String,
-				Resolver: schema.ParentColumnResolver("arn"),
+				Name:       "bucket_arn",
```

Comment:
Doesn't this mean we always save a single rule per bucket in the DB? 

Can we have multiple rules per https://docs.aws.amazon.com/AmazonS3/latest/userguide/ManageCorsUsing.html#cors-example-1?

Reply:
Added `ID` to PK in 6f45f34aff6ef778f06dd815fcbd81247237c429